### PR TITLE
Implemented ability to setup upload translations flags for crowdin

### PR DESCRIPTION
**Pull request by ghost**
Added ability to configure crowdin upload translation flags like there https://support.crowdin.com/configuration-file/#translations-upload

### CODE REVIEW

File: lib/Serge/Sync/Plugin/TranslationService/crowdin.pm [hunk 1]
```diff
@@ -76,7 +86,17 @@ sub push_ts {
...
+        my @flags = ();
+        if ($self->{data}->{import_duplicates}) {
+            push @flags, q/--import-duplicates/
+        }
+        if ($self->{data}->{import_eq_suggestions}) {
+            push @flags, q/--import-eq-suggestions/
+        }
+        if ($self->{data}->{auto_approve_imported}) {
+            push @flags, q/--auto-approve-imported/
+        }
```
Comment (dragosv) [inline lib/Serge/Sync/Plugin/TranslationService/crowdin.pm:98]: I would like to have the command constructed explicitly, without changing run_crowdin_cli. This is how the other plugins are implemented. I would do this by 

```perl
my $action = upload translations 
if ($self->{data}->{auto_approve_imported}) {
$action .= '--import-duplicates'.
}
```

See also the mojito implementation.

File: lib/Serge/Sync/Plugin/TranslationService/crowdin.pm [hunk 2]
```diff
@@ -34,15 +37,21 @@ sub validate_data {
...
 
     $self->{data}->{upload_translations} = 1 unless defined $self->{data}->{upload_translations};
+    $self->{data}->{import_duplicates} = 0 unless defined $self->{data}->{import_duplicates};
+    $self->{data}->{import_eq_suggestions} = 0 unless defined $self->{data}->{import_eq_suggestions};
+    $self->{data}->{auto_approve_imported} = 0 unless defined $self->{data}->{auto_approve_imported};
 }
 
 sub run_crowdin_cli {
-    my ($self, $action, $langs, $capture) = @_;
+    my ($self, $action, $langs, $flags, $capture) = @_;
```
Comment (dragosv) [inline lib/Serge/Sync/Plugin/TranslationService/crowdin.pm:54]: See my comment below regarding avoiding the flags usage. The action in should be passed with all the flags.

**Review by dragosv — CHANGES_REQUESTED** [review]
Also, keep in mind that the translation services plugins are a copy of the master that lives at https://github.com/dragosv/serge-plugins. Please create a pull request into that repo.

**Comment by ghost** [comment]
see PR https://github.com/dragosv/serge-plugins/pull/10

**Pull request closed without merging**